src/views/AddTenant/addTenant.js

```diff
@@ -40,6 +41,7 @@ export const AddTenant = () => {
   const [propertyOptions, setPropertyOptions] = useState([]);
```

Comment:
Can you remove the lease object's .required in the validation schema on line 28? That will allow us to have unhoused tenants in the system which we'll need.

Reply:
Let's also remove the Occupants .required property